From aa501b29ee10da4ab6f36f4e95ab2e2eba847285 Mon Sep 17 00:00:00 2001
From: Algea Cao <algea.cao@rock-chips.com>
Date: Wed, 11 Nov 2020 19:15:28 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3568: Add HCLK_VOP for hdmi

If hdmi doesn't enable HCLK_VOP, clock reference
count will be abnormal and may cause system stuck.

Signed-off-by: Algea Cao <algea.cao@rock-chips.com>
Change-Id: Ibe01d029d1ae69f9f32d4cd0eb149bf72a662381
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index d6c29647ccae..c85029159058 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -1227,8 +1227,9 @@
 		clocks = <&cru PCLK_HDMI_HOST>,
 			 <&cru CLK_HDMI_SFR>,
 			 <&cru CLK_HDMI_CEC>,
-			 <&pmucru PLL_HPLL>;
-		clock-names = "iahb", "isfr", "cec", "ref";
+			 <&pmucru PLL_HPLL>,
+			 <&cru HCLK_VOP>;
+		clock-names = "iahb", "isfr", "cec", "ref", "hclk";
 		power-domains = <&power RK3568_PD_VO>;
 		reg-io-width = <4>;
 		rockchip,grf = <&grf>;
-- 
2.35.3

